src/hooks/aelin/useAelinCreatePool.tsx

```diff
@@ -355,29 +386,36 @@ export default function useAelinCreatePool(chainId: ChainsValues) {
   }
   const handleCreatePool = async () => {
     const {
-      dealDeadLineDuration,
-      investmentDeadLineDuration,
-      investmentToken,
-      poolAddresses,
-      poolAddressesAmounts,
-      poolCap,
-      poolName,
-      poolSymbol,
+      allowListAddresses,
+      allowListAmounts,
+      duration,
+      name,
+      nftCollectionRules,
+      purchaseDuration,
+      purchaseToken,
+      purchaseTokenCap,
       sponsorFee,
+      symbol,
     } = await parseValuesToCreatePool(createPoolState as CreatePoolStateComplete)
 
+    // Run, create a pool and review the values
+    console.log('nftCollectionRules: ', nftCollectionRules)
```

Comment:
Review carefully these values before merging into the feature branch

Reply:
yeah, we should add a build rule or smth